sanitizer/sanitizer.js

```diff
@@ -713,7 +713,7 @@ var getDefaultCSSWhiteList = lib.getDefaultWhiteList;
 
 function getDefaultWhiteList$1() {
   return {
-    a: ["target", "href", "title"],
+    a: ["target", "href", "title", "download"],
```

Comment:
You shouldn't modify this here as this file is auto-generated, and risks to be overwritten.
You'd want to look at that function: https://github.com/blockchain-certificates/blockcerts-verifier/blob/master/sanitizer/index.js#L14